pkg/constant/fast_confirmation_multiplier.go

```diff
@@ -0,0 +1,55 @@
+package constant
+
+import (
+	sdkmath "cosmossdk.io/math"
+
+	"github.com/zeta-chain/node/pkg/chains"
+)
+
+var (
+	// DefaultInboundFastConfirmationLiquidityMultiplier is the default ZRC20 liquidity cap multiplier for inbound fast confirmation
+	DefaultInboundFastConfirmationLiquidityMultiplier = sdkmath.LegacyMustNewDecFromStr("0.0001")
+
+	// InboundFastConfirmationLiquidityMultiplierMap maps chainID to ZRC20 liquidity cap multiplier for inbound fast confirmation.
+	// Fast inbound confirmation is enabled only for chains explicitly listed in this map.
+	InboundFastConfirmationLiquidityMultiplierMap = map[int64]sdkmath.LegacyDec{
```

Comment:
@CharlieMc0 here I defined a static map `chain_id -> liquidity_cap_multiplier` in `zetaclientd` binary. I place `0.0001` as a placeholder for all chains. Please let me know once you get concrete numbers for each chain id.

Reply:
What happens if we add a new chain without a corresponding zetaclientd release? Does it inherit the default automatically?

Reply:
@CharlieMc0 new chain needs to be added to the map explicitly.

Reply:
Hi @CharlieMc0 and @lumtis . I'm going use the default value for newly added chains and remove the whole map. Operation wise, it just needs to update the chain params for newly added chain to enable `FAST` confirmation.

Reply:
> Also, we should support only EVM and Bitcoin chain because other chains don't use confirmation count.

Then you just specify same fast and safe values for observation

Reply:
refactor done in commit https://github.com/zeta-chain/node/pull/3551/commits/6596863f4b5045e05799cfd06e5e40543ca2c90f